frontend/src/editor/webvtt_export.tsx

```diff
@@ -24,7 +24,7 @@ export function WebvttExportModal({
   const [format, setFormat] = useState('vtt' as SubtitleFormat);
 
   return (
-    <Modal {...props} onClose={onClose} label="Export as WebVTT">
+    <Modal {...props} onClose={onClose} label="Export as">
```

Comment:
Maybe just `Export`?

Reply:
I am not that tied to this title, but `Export as` seems a common title for dialogs that let you choose the export format. 
Furthermore it acts as a title for the format dropdown, which I like.

Reply:
then I would at least add ...

Reply:
`…` is even better